presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaMetadata.java

```diff
@@ -308,6 +309,11 @@ private boolean isTablesEnumeratingTable(SchemaTableName schemaTableName)
         return ImmutableSet.of(TABLE_COLUMNS, TABLE_VIEWS, TABLE_TABLES, TABLE_TABLE_PRIVILEGES).contains(schemaTableName);
     }
 
+    private boolean isColumnsEnumeratingTable(SchemaTableName schemaTableName)
+    {
+        return TABLE_COLUMNS.equals(schemaTableName);
```

Comment:
not only `columns` table, but likely `table_privileges` too?

Reply:
It doesn't have `column` column

Reply:
It doesn't, but it needs to be filtered by `table_name`. Will it be?